gems/sorbet-runtime/lib/types/types/type_parameter.rb

```diff
@@ -3,11 +3,30 @@
 
 module T::Types
   class TypeParameter < Base
+    module Private
+      @pool = {}
+
+      def self.cached_entry(name)
+        @pool[name]
+      end
+
+      def self.set_entry_for(name, type)
+        @pool[name] = type
+      end
+    end
+
     def initialize(name)
       raise ArgumentError.new("not a symbol: #{name}") unless name.is_a?(Symbol)
       @name = name
     end
 
+    def self.make(name)
```

Comment:
Should we add an entry to the RBI in the payload for this new method?

Reply:
Sure, we can do that.